synchronous-response-springboot/src/main/java/org/example/camunda/process/solution/worker/CalculateSomethingWorker.java

```diff
@@ -13,15 +13,15 @@ public class CalculateSomethingWorker {
   
   private static final Logger LOG = LoggerFactory.getLogger(CalculateSomethingWorker.class);
 
-
   @JobWorker
   public Map<String, Object> calculateSomething() {
-    try {
-      LOG.info("Cause a 5 seconds delay");
-      Thread.sleep(5000);
-    } catch (InterruptedException e) {
-      LOG.error("Thread sleep got interrupted: {}", e.getMessage());
-    }
+    // To see a real delay on the web page, uncomment the following lines    
```

Comment:
On which web page? Sorry for being nasty on the details ;-)

Reply:
It was too early in the morning... It's just a request/response that gets delayed